### PR TITLE
Demo using Github Actions for flake8

### PR DESCRIPTION
**Pull request by dralley**
[noissue]

### CODE REVIEW

**Comment by dralley** [comment]
Doesn't take effect until it gets merged to the repo, so take a look at my branch: https://github.com/dralley/pulpcore/actions/runs/32015445

**Comment by fabricio-aguiar** [comment]
> It wouldn't show up here in this org until after being merged, so take a look at my branch: https://github.com/dralley/pulpcore/runs/412028305?check_suite_focus=true
> 
> I also temporarily added a deliberate flake lint failure

pretty cool @dralley 
thanks for this demo

File: .github/workflows/ci.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+name: Pulp CI Lints
+
+on: [push, pull_request]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
```
Comment (fabricio-aguiar) [inline .github/workflows/ci.yaml]: this [runs-on](https://help.github.com/en/actions/automating-your-workflow-with-github-actions/workflow-syntax-for-github-actions#jobsjob_idruns-on) really caught my attention!

**Comment by fabricio-aguiar** [comment]
Just for reference:
[foreman-ansible-modules](https://github.com/theforeman/foreman-ansible-modules) in: 
- [Travis](https://travis-ci.org/theforeman/foreman-ansible-modules/builds)
- [GH actions](https://github.com/theforeman/foreman-ansible-modules/actions?query=workflow%3ACI)

**Pull request closed without merging**